lib/src/util/component_usage_migrator.dart

```diff
@@ -433,6 +433,23 @@ abstract class ComponentUsageMigrator with ClassSuggestor {
     yieldPatch('', start, end);
   }
 
+  /// Yields a patch that adds a given [child] to a usage.
+  void yieldAddChildPatch(FluentComponentUsage usage, String child) {
+    final int start;
+
+    if (!child.endsWith(',')) {
```

Comment:
#nit Do we need this `endsWith` check, or could we just assume the input doesn't have a trailing comma and always add one?

Reply:
Good point, that sounds great to me!